src/cli/ionic-config.ts

```diff
@@ -0,0 +1,45 @@
+import { readJSON, writeJSON, mkdirp } from '@ionic/utils-fs';
+
+import * as os from 'os';
+import * as path from 'path';
+
+import { uuidv4 } from './telemetry/helpers';
+
+export const CONFIG_FILE = 'config.json';
+export const DEFAULT_CONFIG_DIRECTORY = path.resolve(os.homedir(), '.ionic', CONFIG_FILE);
+
+export interface SystemConfig {
```

Comment:
Consider renaming this to `TelemetryConfig`, as `SystemConfig` may be easily conflated with other parts of the 'system' (e.g. is this a compiler configuration, a build configuration, etc.)
```suggestion
export interface TelemetryConfig {
```